src/Components/OATHeader/OATHeader.tsx

```diff
@@ -77,6 +91,42 @@ const OATHeader = ({ elements, onImportClick }: OATHeaderProps) => {
         }
     ];
 
+    useEffect(() => {
+        const newFiles = [];
+        acceptedFiles.forEach((sF) => {
+            if (sF.type === 'application/json') {
+                newFiles.push(sF);
+            }
+        });
+        handleFileListChanged(newFiles);
+    }, [acceptedFiles]);
+
+    const handleFileListChanged = async (files: Array<File>) => {
+        const items = [];
+        if (files.length > 0) {
+            for (const current of files) {
+                const newItem = {
+                    name: current.name,
+                    size: prettyBytes(current.size),
+                    status: FileUploadStatus.Uploading
+                } as IFileItem;
+                try {
+                    const content = await current.text();
+                    newItem.content = JSON.parse(content);
+                } catch (error) {
+                    console.log(Error(error));
```

Comment:
Why do we need to wrap this is an error?

Reply:
Took that from the example class, and Milton is actually going to be working In the Error Handling Component, so we should be using that instead eventually

Reply:
Can you please provide a link to the example class?

I would've expected a read/parse error to result in `error` already being of type `Error`. I.e. can't we just `console.log(error);`?